src/Microsoft.ML.Core/Data/ModelLoading.cs

```diff
@@ -21,6 +21,23 @@ internal sealed partial class ModelLoadContext : IDisposable
         public const string ModelStreamName = "Model.key";
         internal const string NameBinary = "Model.bin";
 
+        /// <summary>
+        /// Returns the new assembly name to maintain backward compatibility.
+        /// </summary>
+        private string ForwardedLoaderAssemblyName
```

Comment:
>ForwardedLoaderAssemblyName [](start = 23, length = 27)

my $0.02 makes sense to do this post V1; i don't see it necessary to drag this legacy in the codebase.

Reply:
That's reasonable, but there was a [discussion here](https://github.com/dotnet/machinelearning/pull/2792#issuecomment-468319048) where we decided that this sort of approach would be preferable from a practical point of view.

---
In reply to: [261829875](https://github.com/dotnet/machinelearning/pull/2800#discussion_r261829875) [](ancestors = 261829875)